> Just need to adjust the imports and run isort.

I've tried adding these imports, but a) when I run hass, the extra sensors do now show up but there are not any errors reported either, b) the commit checks fail.

What I tried:
```
-from pyvesync.vesyncbasedevice import VeSyncBaseDevice
+from pyvesync.vesyncfan import VeSyncAirBypass
+from pyvesync.vesyncoutlet import VeSyncOutlet
+from pyvesync.vesyncswitch import VeSyncSwitch
```

The commit check fails:
```
************* Module homeassistant.components.vesync.sensor
homeassistant/components/vesync/sensor.py:8:0: E0401: Unable to import 'pyvesync.vesyncfan' (import-error)
homeassistant/components/vesync/sensor.py:9:0: E0401: Unable to import 'pyvesync.vesyncoutlet' (import-error)
homeassistant/components/vesync/sensor.py:10:0: E0401: Unable to import 'pyvesync.vesyncswitch' (import-error)
```

--- a/homeassistant/components/vesync/sensor.py
+++ b/homeassistant/components/vesync/sensor.py
@@ -1,4 +1,5 @@
 """Support for power & energy sensors for VeSync outlets."""
+from __future__ import annotations
 from collections.abc import Callable
 from dataclasses import dataclass
 import logging

--- a/homeassistant/components/vesync/sensor.py
+++ b/homeassistant/components/vesync/sensor.py
@@ -43,8 +43,12 @@ class VeSyncSensorEntityDescription(
 ):
     """Describe VeSync sensor entity."""
 
-    exists_fn: Callable[[VeSyncBaseDevice], bool] = lambda _: True
-    update_fn: Callable[[VeSyncBaseDevice], None] = lambda _: None
+    exists_fn: Callable[
+        [VeSyncAirBypass | VeSyncOutlet | VeSyncSwitch], bool
+    ] = lambda _: True
+    update_fn: Callable[
+        [VeSyncAirBypass | VeSyncOutlet | VeSyncSwitch], None
+    ] = lambda _: None
 
 
 def update_energy(device):

--- a/homeassistant/components/vesync/sensor.py
+++ b/homeassistant/components/vesync/sensor.py
@@ -157,7 +157,7 @@ class VeSyncSensorEntity(VeSyncBaseEntity, SensorEntity):
 
     def __init__(
         self,
-        device: VeSyncBaseDevice,
+        device: VeSyncAirBypass | VeSyncOutlet | VeSyncSwitch,
         description: VeSyncSensorEntityDescription,
     ) -> None:
         """Initialize the VeSync outlet device."""

